packages/bem-react-scripts/config/webpack.config.dev.js

```diff
@@ -74,7 +77,7 @@ module.exports = {
     // This does not produce a real file. It's just the virtual path that is
     // served by WebpackDevServer in development. This is the JS bundle
     // containing code from all our entry points, and the Webpack runtime.
-    filename: 'static/js/bundle.js',
+    filename: setName ? `static/js/${setName}.bundle.js` : `static/js/[name].bundle.js`,
```

Comment:
this is still differ from what it was

Reply:
now if not setName then the build will work as usual

Reply:
he asks about `[name]` notation. May be it somehow can handle `setName`